test/test_autograd.py

```diff
@@ -5957,6 +5957,23 @@ def where(cond, x, y):
         gradcheck(where, [cond, x, y], raise_exception=True)
         gradgradcheck(where, [cond, x, y], [torch.randn(5, 5, 5, device=device)])
 
+    def test_where_scalar(self, device):
```

Comment:
I think we should just add these tests with other tests for `torch.where` [here](https://github.com/pytorch/pytorch/blob/45c5bac87049a9a0593d4d5f060cc8d4b0f83db8/torch/testing/_internal/common_methods_invocations.py#L931-L935)

Reply:
I don't think it will work in `common_method_invocations` as this variant is functional only.